azurestack/data_source_network_interface.go

```diff
@@ -181,78 +181,62 @@ func dataSourceArmNetworkInterfaceRead(d *schema.ResourceData, meta interface{})
 
 	d.SetId(*resp.ID)
 
-	iface := *resp.InterfacePropertiesFormat
-
-	d.Set("mac_address", iface.MacAddress)
+	d.Set("name", resp.Name)
+	d.Set("resource_group_name", resGroup)
+	if location := resp.Location; location != nil {
+		d.Set("location", azureStackNormalizeLocation(*location))
+	}
 
-	if iface.IPConfigurations != nil && len(*iface.IPConfigurations) > 0 {
-		configs := *iface.IPConfigurations
+	if iface := resp.InterfacePropertiesFormat; iface != nil {
+		d.Set("mac_address", iface.MacAddress)
+		d.Set("enable_ip_forwarding", iface.EnableIPForwarding)
 
-		if configs[0].InterfaceIPConfigurationPropertiesFormat != nil {
-			privateIPAddress := configs[0].InterfaceIPConfigurationPropertiesFormat.PrivateIPAddress
-			d.Set("private_ip_address", *privateIPAddress)
-		}
+		if iface.IPConfigurations != nil && len(*iface.IPConfigurations) > 0 {
+			configs := *iface.IPConfigurations
 
-		addresses := make([]interface{}, 0)
-		for _, config := range configs {
-			if config.InterfaceIPConfigurationPropertiesFormat != nil {
-				addresses = append(addresses, *config.InterfaceIPConfigurationPropertiesFormat.PrivateIPAddress)
+			if configs[0].InterfaceIPConfigurationPropertiesFormat != nil {
+				privateIPAddress := configs[0].InterfaceIPConfigurationPropertiesFormat.PrivateIPAddress
+				d.Set("private_ip_address", *privateIPAddress)
```

Comment:
since this is a top level simple object I think we should be able to ditch the `*` here?